src/main/groovy/pl/droidsonroids/gradle/pitest/PitestPlugin.groovy

```diff
@@ -67,7 +67,7 @@ class PitestPlugin implements Plugin<Project> {
     private static final String PITEST_JUNIT5_PLUGIN_NAME = "junit5"
     private final static List<String> DYNAMIC_LIBRARY_EXTENSIONS = ['so', 'dll', 'dylib']
     private final static List<String> DEFAULT_FILE_EXTENSIONS_TO_FILTER_FROM_CLASSPATH = ['pom'] + DYNAMIC_LIBRARY_EXTENSIONS
-    private final static Pattern MOCKABLE_ANDROID_JAR_PATH_PATTERN = Pattern.compile('.*/platforms/android-.*/android.jar$')
+    private final static Pattern MOCKABLE_ANDROID_JAR_PATH_PATTERN = Pattern.compile(".*\\${File.separator}platforms${File.separator}android-.*${File.separator}android.jar\$")
```

Comment:
It doesn't look good:
```suggestion
    private final static Pattern MOCKABLE_ANDROID_JAR_PATH_PATTERN = Pattern.compile(".*${File.separator}platforms${File.separator}android-.*${File.separator}android.jar\$")
```
 ?

Reply:
https://gwc-experiment.appspot.com/?g=groovy_3_0&codez=eNptjEEKwjAQRfc9RSiFpoKTE7h17QHcjM1UU5JOmExFEO9uF1UK-lcfHu95Gkw2BzPiHWHWEEHoSg84oSrJBD2nHCLZGnbN87g8KJRRUFleOaIOLKn8EJy8cPD7P9KKYEQ5N3VXVVgKiZoMCbW_kdh2YHbftvu0LihuI7fdKhS7RMxmb7WWSRY

Reply:
> and it failed with:

Sure, on windows is `\p` which could be treated as a "control sequence". On Linux is not a problem.

Reply:
It seems that even Groovy standard library itself does not support Windows 😅 
```
    !project.tasks.getByName('pitestRelease').additionalClasspath.getAsPath().contains('android.jar')
     |       |     |                          |                   |
     |       |     task ':pitestRelease'      file collection     java.util.regex.PatternSyntaxException: Unexpected internal error near index 1
```
I've tried several ways to examine a FileCollection but all them fail with the same exception on Appveyor Windows machine only.